cv.md

```diff
@@ -0,0 +1,53 @@
+# Aliaksandr Isachanka
+
+## Contact Info: 
+
+* _Email:_ 
+    * foxtrot2609@gmail.com
+
+* _Phone and Messenger App:_ 
+    * +375 29 776 79 85 (viber, telegram)
+    * +375 25 670 97 50 (whatsapp, telegram)
+    * Telegram: @foxtrot26 or @foxtrot2609
+
+* _Social networks:_
+    * [Personal page on Facebook](https://www.facebook.com/people/Alexandr-Isachenko/100004995227722)
+    * [Personal page on LinkedIn](https://www.linkedin.com/in/alexandr-isachenko-5548b39a)
+    
+
+## Summary:
+Started by ownself learning the basics of HTML and CSS in 2010, as a tool to create useful new products and realize own creativity. It was my hobby, not my main job.  
+
+Today I decided to devote all my time to what I really like - Frontend development. 
+
+I was always able to find reliable information and solutions for different issues. I am capable of completing projects successfully. For example, my first experience in launching website - [meteotur.ru](http://www.meteotur.ru/index.php).
+
+In the short term, my goal is to improve my knowledge, skills and achievements to the level required for a Junior Developer. 
+
+In the long term I want to get job in growing IT company, to take a part in projects and as result of my own development to lead one of them.
+    
+## Skills 
+* basic skills in HTML5, CSS3, JavaScript, Git & Github, Markdown
+* basic skills in copywriting and SEO optimization (own blog [go4gu.ru](https://go4gu.ru/))
+* photography skills, confident work with Photoshop and other graphic editors
+* basic skills in work with video content and video editors
+
+## Code examples 
+* [GitHub branch with latest code example](https://github.com/foxtrot2609/Example_of_last_work)
+
+## Experience 
+* [meteotur.ru](http://www.meteotur.ru/index.php) - my first experience in HTML and CSS
```

Comment:
Would be great if you will have more detailed description what exact was done by you for those project. E.g. 
Make up html and css from scratch 
Made changes to existing theme
Set-up wrodpress , configure and modify plugins for wrodpress